ext/curl/curl_private.h

```diff
@@ -75,6 +75,9 @@ typedef struct {
 	php_curl_callback *progress;
 #if LIBCURL_VERSION_NUM >= 0x072000
 	php_curl_callback  *xferinfo;
```

Comment:
Note for self/refactoring, the php_curl_callback could all be replace with FCCs and the new FCC API I introduced as it keeps track of closures now.